ironfish/src/blockchain/blockchain.ts

```diff
@@ -959,43 +947,25 @@ export class Blockchain<
       [this.notes.counter, this.notes.leaves, this.notes.nodes],
```

Comment:
I think the comment for `addNote` should remove references to `looseNotes`. Also, notes can't come in any order now (unrelated to this PR).